sql-db/narayana-transactions/src/test/resources/mysql.properties

```diff
@@ -1,5 +1,4 @@
 quarkus.datasource.db-kind=mysql
-quarkus.hibernate-orm.dialect=org.hibernate.dialect.MariaDB102Dialect
```

Comment:
Why we are removing the dialect?

Reply:
Because using MariaDB dialect with MySQL is nonsense. For one, it breaks with sequences as there are no sequences (well in a sense, yes there are with InnoDB)